### PR TITLE
Adds nixpkgs to installation section

### PR DESCRIPTION
**Pull request by legendofmiracles**
Hope it's ok that i packaged it :)

### CODE REVIEW

**Comment by Freddson** [comment]
lgtm

**Review by TorchedSammy — APPROVED** [review]
lgtm

**Review by Freddson — APPROVED** [review]
lgtm even more

File: README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ $ mv tmpmail ~/bin/
 $ yay -S tmpmail-git
 ```
 
+### Nixpkgs
+`tmpmail` is also avaible in the [nix package collection (only unstable currently)](https://search.nixos.org/packages?channel=unstable&show=tmpmail&from=0&size=50&sort=relevance&query=tmpmail), which is maintained by [legendofmiracles](https://github.com/legendofmiracles)
```
Comment (TorchedSammy) [inline README.md:42]: didnt notice this, typo here

**Review by TorchedSammy — CHANGES_REQUESTED** [review]

**Review by Freddson — APPROVED** [review]
now that is the real lgtm

**Review by TorchedSammy — APPROVED** [review]

**Review by legendofmiracles — COMMENTED** [review]
LGTM

**Comment by breadtard** [comment]
what does lgtm mean

**Comment by TorchedSammy** [comment]
@breadtard looks good to me

**Pull request merged**